fk j

```diff
@@ -0,0 +1,15 @@
+  [31morigin/HEAD[m -> origin/develop
```

Comment:
What is the purpose of this file? If it is redundant add to .gitignore

Reply:
I'm not sure, it seems to be a incomplete redundant file generated by git. I will remove it completely and add it to .gitignore as well.
Thanks for noting.

Reply:
Removed in this commit https://github.com/TIA-Lab/tiatoolbox/commit/b7e39b24199f9e1270b0e27da304e71427c31582